packages/wdio-selenium-standalone-service/src/@types/webdriverio.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module WebdriverIO {
+    interface Config {
+        watch?: boolean
```

Comment:
Can we move this to the original typings?

Reply:
@mgrybyk  added this in his commit. Should i remove it?

Reply:
The `watch` property in the config isn't something that is specific to the service, so I would say yes.

Reply:
We should think on how to keep it internal only and don't expose to the user.
At least disallow to define it in the config

Reply:
Otherwise it might confuse a user that the option does nothing
https://github.com/webdriverio/webdriverio/blob/e34108325ef1e0d2b8e2306fe407b5d934169777/packages/wdio-config/src/lib/ConfigParser.js#L73

Reply:
We can also define it in webdriverio tpl config like this `readonly watch?: never` and add a comment that it's used for internal purposes only.
@christian-bromann

Reply:
sounds good to me

Reply:
Second one, I guess

Reply:
Yes, this is an internal flag and only set if you run:

```sh
$ npx wdio wdio.conf.js --watch
```

as the `watch` flag will be merged into the config.